docs/ipr/scripting/RNA_Expression_Metrics.ipynb

```diff
@@ -0,0 +1,4089 @@
+              "      <td>-9.9658</td>\n",
+              "      <td>-9.9658</td>\n",
+              "      <td>-9.9658</td>\n",
+              "      <td>-9.9658</td>\n",
+              "      <td>-9.9658</td>\n",
+              "      <td>-9.9658</td>\n",
+              "      <td>-9.9658</td>\n",
+              "      <td>-9.9658</td>\n",
+              "      <td>-9.9658</td>\n",
+              "      <td>-9.9658</td>\n",
+              "      <td>-9.9658</td>\n",
+              "      <td>-9.9658</td>\n",
+              "      <td>-9.9658</td>\n",
+              "      <td>-9.9658</td>\n",
+              "      <td>-9.9658</td>\n",
+              "    </tr>\n",
+              "  </tbody>\n",
+              "</table>\n",
+              "<p>60498 rows × 45 columns</p>\n",
+              "</div>"
+            ],
+            "text/plain": [
+              "                    TCGA-ZH-A8Y5-01  ...  TCGA-W5-AA2X-11\n",
+              "gene                                 ...                 \n",
+              "ENSG00000242268.2           -9.9658  ...          -9.9658\n",
+              "ENSG00000259041.1           -9.9658  ...          -9.9658\n",
+              "ENSG00000270112.3           -9.9658  ...          -9.9658\n",
+              "ENSG00000167578.16           4.0575  ...           3.8808\n",
+              "ENSG00000278814.1           -9.9658  ...          -9.9658\n",
+              "...                             ...  ...              ...\n",
+              "ENSG00000273233.1           -9.9658  ...          -9.9658\n",
+              "ENSG00000105063.18           5.1157  ...           3.4778\n",
+              "ENSG00000231119.2           -4.2934  ...          -2.9324\n",
+              "ENSG00000280861.1           -9.9658  ...          -9.9658\n",
+              "ENSG00000181518.3           -9.9658  ...          -9.9658\n",
+              "\n",
+              "[60498 rows x 45 columns]"
+            ]
+          },
+          "metadata": {
+            "tags": []
+          },
+          "execution_count": 44
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "HBCmQl9OTLVM"
+      },
+      "source": [
+        "Now that we have our matrix we are ready to start calculating metrics"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "i6Ptf4p9TXs6"
+      },
+      "source": [
+        "## Calculating Metrics\n",
+        "\n",
+        "The input for an analysis where a patient sample is being compared to the TCGA cohort of interest would typically be an expression matrix using the same normalization method as the comparator cohort. Note: Differences in library preparation, sequencing and bioinformatics pipelines can lead to variation in expression quantification. It is important to understand the impact these differences have on expression quantification before comparing expression values that are produced using different pipelines.\n",
```

Comment:
maybe remove 'Note:'

Reply:
I changed this to follow the same note format as elsewhere